### ISSUE TITLE
(sacrilege warning) remove 'suicide' from root level keybindings

### ISSUE THREAD

**Issue by I-am-Erk** [CleverRaven/Cataclysm-DDA#59679]
### Is your feature request related to a problem? Please describe.

This is part of a larger effort to identify places where our "top level" keybindings have been mapped to things that players do not need rapid, single-keypress access for. This is a discussion: all these issues are valid places to raise concerns about reasons a player should have immediate single-key access to this function.

Long and short: nobody needs an immediate access key for suicide. Despite q-y-y being a meme, it's not a good use of a top level key.

### Solution you would like.

Move "suicide this character" to the ESC menu to free that binding for another use

### Describe alternatives you have considered.

We could also move "S"ave and quit off the default keybindings, but that is a more contentious one.

It would be nice if there were some semi-humorous ways to get the same effect such as attempting to drink bleach.

**Comment by NetSysFire** [comment]
I approve of this and think this maybe even should get done in time for 0.G, since 0.G may attract new players.
"Alright lets quit for now and call it a day. Suicide? What? Is this how you quit?"
A quick solution would be to move Save and quit to this and make ctrl+s just a regular quicksave or something.

**Comment by I-am-Erk** [comment]
Yeah, that is a pretty nice move actually. I don't remember what lower-case `q` is bound to but another option might be to do `q`uicksave, `Q`uit and save

Actually from loading my game it seems `q` is unbound, probably because of how dangerous `Q` is, so that should be a pretty easy fix. Then that frees up `shift+S` for some as-yet undetermined use (I would like it to become Sleep/Wait, or maybe Fighting Style, but I don't want to remap too much stuff in one pass)

**Comment by pjf** [comment]
If we could have the same functionality, but call it "abandon character", I would personally appreciate it. I've lost too many friends iRL, and somehow the wording of the `Q` keybinding still hits me every time.

**Comment by liminalRoar** [comment]
`Shift+S` could be Spell, hehe, since that is undedicated although i suppose that has no use outside of Magiclysm

I don't really see a need for any dedicated default bindings for quitting and think it's fine on the ESC menu, at least if we have better, frequent use cases for the "Q"s and "S".

**Comment by NetSysFire** [comment]
>Shift+S could be Spell, hehe, since that is undedicated although i suppose that has no use outside of Magiclysm

Some mutations have spells. E.g the leap spell when you have frog legs.

**Comment by liminalRoar** [comment]
We could remap some of the `SHIFT+number key`-bindings to easier things. like sleep, construction, disassemble (though that menu needs to die imo)

**Comment by keampe** [comment]
This is an excellent idea, but would also like to second the idea of renaming it to something else like "Abandon Character", for the same reason.

   - Shane

**Comment by Fris0uman** [comment]
shift + q is now unbound
